pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -193,6 +194,7 @@ let
           hash = "sha256-lDWX69YENZFMu7pyBmavUZaalGvFqbHSHfkwkzmDQaY=";
         };
       });
+
       # Ceph does not support `cryptography` > 40 yet:
       # * https://github.com/NixOS/nixpkgs/pull/281858#issuecomment-1899358602
       # * Upstream issue: https://tracker.ceph.com/issues/63529
```

Comment:
```suggestion
      # * Upstream issue: https://tracker.ceph.com/issues/64213
```

Reply:
I've merged that link with the other one above about `bcrypt`, where both issues are linked.